deployment/ansible/group_vars/all

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
It feels a little weird to me to template this file, since a lot of those variables won't ever be customized (e.g. the OTP version, jarfile, repo, etc). I'm thinking it would make more sense to template the development group_vars file instead -- put the secrets in there, and have the all file contain the things that are more or less static.